### PR TITLE
Feature/cmn 398 renew loans

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to issue

n/a

#### Description

- button disable are not supposed to be black
- renew is (apparently) not with faust but with loanid
- remove unused mapper

#### Screenshot of the result

disabled knap: 

image

#### Checklist

- [ ] My complies with [our coding guidelines](../docs/code_guidelines.md).
- [ ] My code passes our static analysis suite. If not then I have added a comment explaining why this change should be exempt from the code standards and process.
- [ ] My code passes our continuous integration process. If not then I have added a comment explaining why this change should be exempt from the code standards and process.

#### Additional comments or questions

If you have any further comments or questions for the reviewer them please add them here.

### CODE REVIEW

**Review by jeppekroghitk — APPROVED** [review]

File: src/apps/loan-list/materials/selectable-material/selectable-material.tsx [hunk 1]
```diff
@@ -15,8 +14,8 @@ import StatusMessage from "./StatusMessage";
 interface SelectableMaterialProps {
   loan: LoanType;
   disabled?: boolean;
-  materialsToRenew: FaustId[];
-  onChecked?: (faust: FaustId) => void;
+  materialsToRenew: number[];
+  onChecked?: (loanId: number) => void;
```
Comment (spaceo) [inline src/apps/loan-list/materials/selectable-material/selectable-material.tsx:18]: I think we could put the loan id along the other knowns id's by creating a `LoanId` type in /core/utils/types/ids. It is already used in multiple places and we can benefit from it when the functionality with loan id expands.

**Review by spaceo — CHANGES_REQUESTED** [review]
Looks awesomeness. One comment...

File: src/apps/loan-list/materials/selectable-material/selectable-material.tsx (hunk 1 above)
Comment (sinejespersen) [reply src/apps/loan-list/materials/selectable-material/selectable-material.tsx:18]: export type LoanId = number;  ?

Comment (spaceo) [reply src/apps/loan-list/materials/selectable-material/selectable-material.tsx:18]: Yes same as FaustId:
```
export type FaustId = `${number}`;

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**